languages/ruby/exercises/concept/booleans/.meta/example.rb

```diff
@@ -0,0 +1,15 @@
+class RidePass
+  MINIMUM_HEIGHT = 120
+
+  def issue(height)
```

Comment:
```suggestion
  def issue!(height)
```

Reply:
Why the "bang" here if nothing is changed in place?  State is not modified.

Reply:
Perhaps the confusion is because the class name is `RidePass` rather than `RidePassIssuer`?

Reply:
I think the confusion stems more from it not modifying the state.  If it is issued and the criterion is height, then it could be approved or denied here, and we would issue the pass, presumably, or revoke the pass?  If it modifies the state of the pass then a bang is appropriate as a warning that this modifies state.  If the word implies a change, then, like other Ruby methods where the name is enough (or is deemed enough of) a warning that state will change, then the bang is left off.  An example of this is the `delete` method that can be found.  No "bang" required.

Reply:
Yes, exactly.  But I guess what I failed to imply is that if the class is named `RidePass` then maybe we have been implying that we are manipulating a state of an instance of `RidePass` rather than `RidePass` returning a value to indicate the status of the a ride pass.

Ergo, if it was named `RidePassIssuer` then the progression of `issue` then `revoke` wouldn't seem as much like a state change of an instance of itself.